go/private/actions/library.bzl

```diff
@@ -127,7 +127,8 @@ def emit_library(ctx, go_toolchain, srcs, deps, cgo_object, embed, want_coverage
           importpath = importpath, # The import path for this library
           direct = direct, # The direct depencancies of the library
           transitive = transitive, # The transitive set of go libraries depended on
-          srcs = depset(srcs), # The original sources
+          srcs = depset(srcs), # The sources used for building (contains cgo generated outputs)
```

Comment:
if srcs is the originals, I don't think we need the transformed ones returned in this provider at all, if they are really needed they can be picked out of the GoEmbed provider below.